drivers/regulator/regulator_cp9314.h

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
Are any of these defines needed outside the .c file?
If not they could be moved into the source. file

Reply:
nope, I'll move them.